azkaban-common/src/test/java/azkaban/project/DirectoryFlowLoaderTest.java

```diff
@@ -95,11 +96,14 @@ private static File decompressTarBZ2(InputStream is) throws IOException {
           parent.mkdirs();
         }
 
-        IOUtils.copy(tais, new FileOutputStream(outputFile));
+        FileOutputStream os = new FileOutputStream(outputFile);
+        IOUtils.copy(tais, os);
+        os.close();
```

Comment:
Should be in a separate commit, maybe even PR of its own. If this change even matters..

Reply:
Hi @juhoautio ! It matters, since it allows to release the file descriptors. `IOException` thrown upon the exhaustion of file descriptors is the reason why this test frequently fails.

Reply:
I agree, will update the PR later today